sdk/python/feast/infra/online_stores/dynamodb.py

```diff
@@ -14,12 +14,11 @@
 from datetime import datetime
```

Comment:
Can you change the rest of `table_name` to `table_instance` (where `table_name` is actually an instance of DynamoDB Table object)?

Reply:
done